sdk/examples/wallet/update_alias_output.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! cargo run --example update_alias_output --release
+// In this example we will update the state metadata of an alias output
+// Rename `.env.example` to `.env` first
```

Comment:
Example doc police here ... you have violated regulation 66 ... please come with us for interrogation!

Reply:
Copy pasted from another example where you forgot it :trollface:

Reply:
picket the wrong example then :joy: